cwms-data-api/src/test/resources/cwms/cda/api/timeseriesprofile/ts_profile_ref.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Description",
+  "parameter-list": [
+    "Temp-Water",
+    "Depth"
+  ],
+  "location-id": {
+    "office-id": "SPK",
+    "name": "Test_TSP_Location"
+  },
+  "reference-ts-id": {
+    "office-id": "SWT",
```

Comment:
I don't think it makes sense to mix offices within a single profile.

Reply:
Changed to use the same office in the profile